package.json

```diff
@@ -35,7 +35,7 @@
     "@carbon/charts-react": "~0.41.38",
     "@carbon/icons-react": "~10.26.0",
     "@carbon/themes": "~10.28.0",
-    "@data-driven-forms/carbon-component-mapper": "~2.24.3",
+    "@data-driven-forms/carbon-component-mapper": "~2.24.4",
```

Comment:
This update is needed in order to fix this DDF issue https://github.com/data-driven-forms/react-forms/issues/1048.